### PR TITLE
feat(molecule/dropdownOption): add tokens to autosuggest option

### PR DESCRIPTION
**Pull request by PablitoGS**
## MOLECULE/DROPDOWNOPTION
https://jira.scmspain.com/browse/PI-34511

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context
add tokens to be able to apply design

### Screenshots - Animations
Captura de pantalla 2020-12-22 a las 13 16 21

### CODE REVIEW

File: components/molecule/dropdownOption/src/index.scss [hunk 1]
```diff
@@ -16,13 +17,20 @@ $pl-molecule-dropdown-option: $p-molecule-dropdown-option !default;
 $pr-molecule-dropdown-option: $p-molecule-dropdown-option !default;
 $pt-molecule-dropdown-option: 0 !default;
 $pb-molecule-dropdown-option: 0 !default;
+$ml-molecule-dropdown-option: 0 !default;
+$mr-molecule-dropdown-option: 0 !default;
+$m-molecule-dropdown-option: 0 !default;
+$m-molecule-dropdown-option: 0 !default;
```
Comment (andresin87) [inline components/molecule/dropdownOption/src/index.scss]: duplicated line?

Comment (andresin87) [reply components/molecule/dropdownOption/src/index.scss]: I think you need `mt` and `mb`

Comment (PablitoGS) [reply components/molecule/dropdownOption/src/index.scss]: ups, yes, it's a mistake

**Review by andresin87 — APPROVED** [review]

**Review by klaufel — APPROVED** [review]

**Comment by andresin87** [comment]
Sorry, I figured out that this change can generate bugs in verticals.

**Comment by PablitoGS** [comment]
> Sorry, I figured out that this change can generate bugs in verticals.

@andresin87 why? I think it keeps what we have with the default vars

**Pull request merged**